docs/ci_test/ci_test_overview.md

```diff
@@ -1,6 +1,6 @@
 # CI Tests as GH Workflows in Spellbook
 
-Any time a PR is opened in Spellbook, there are a few GH workflows which automatically run on each commit. The `dbt slim ci` workflow kicks off the continuous integration (CI) tests, which are required to pass in order to prepare for final merge into the main branch. The code which runs the CI workflow can be found [here](https://github.com/duneanalytics/spellbook/blob/main/.github/workflows/dbt_slim_ci.yml), but in general, the steps included are:
+Any time a PR is opened in Spellbook, there are a few GH workflows which automatically run on each commit. The `dbt slim ci` workflow kicks off the continuous integration (CI) tests, which are required to pass in order to prepare for final merge into the main branch. The code which runs the CI workflow can be found [here](.github/workflows/dbt_slim_ci.yml), but in general, the steps included are:
```

Comment:
the reason these were originally built with links is because the relative link wasn't working. if you preview this file, you'll notice the link is broken.

Reply:
interesting. at this time, the preview still returns a 404 error for me. is the expectation that post-merge, the path would update to `main` and work? it'll never work in PR phase?

Reply:
okay, we can give it a shot. worst case, we'll revert if still doesn't work post-merge.